app/filters/link_filter.rb

```diff
@@ -67,9 +72,10 @@ def relativize_local_links!
 
   def rewrite_for_https_support!
     parser.css("iframe,img").each do |iframe|
-      if src = iframe.try(:attributes).try(:[], 'src').try(:value)
-        if matches_https_whitelist?(src) || (src =~ /\Ahttp:\/\// && url_exists?(src))
-          iframe.set_attribute 'src', src.gsub(/\Ahttps?:\/\//, "//")
+      if src = iframe.try(:attributes).try(:[], "src").try(:value)
+        if matches_https_whitelist?(src) ||
+           (src =~ /\Ahttp:\/\// && url_exists?(src))
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.